scripts/winesapos-convert.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
-# Script originally created by @GuestSneezeOSDev
 echo "System is converting ..."
 
-CMD_PACMAN_INSTALL=(sudo pacman --noconfirm -S --needed)
+CMD_PACMAN_INSTALL=(sudo pacman --noconfirm -Syu --needed) # Mohamed: Updated this to Syu instead of S because if the person running this script doesn't have they're system updated, then this'll cause errors.
```

Comment:
Nice catch, thank you!

We should probably have a first step to run `sudo pacman --noconfirm -Syu` and then separately keep the  CMD_PACMAN_INSTALL line the same. I'm not sure how telling the system to upgrade with `-u` and having the `--needed` flag will behave together.